src/QMCHamiltonians/tests/test_force.cpp

```diff
@@ -225,7 +225,7 @@ TEST_CASE("Chiesa Force", "[hamiltonian]")
   std::cout << " Force = " << force.forces << std::endl;
 
   // Unvalidated externally
-  REQUIRE(force.forces[0][0] == Approx(3.186559306));
```

Comment:
Why did this number change?  This doesn't bode well.

Reply:
I agree, but I don't have a specific explanation. I know this is a result of changing from LRHandlerSRCoulomb to EwaldHandler3D, and thus, actual evidence that the handler case needs more testing.